# Homework #1 Data Exploration (Kumar Gaureve Varun - 91131)

# Question 1


## Read the data in R using
Determine whether each of the attributes (columns) are treated as qualitative (categorical) or quantitative (numeric) using R. Explain how you can tell using R.


I'll be using tidyverse set of packages for R to read data and explore the data

In [3]:
library(tidyverse)

Loading tidyverse: ggplot2
Loading tidyverse: tibble
Loading tidyverse: tidyr
Loading tidyverse: readr
Loading tidyverse: purrr
Loading tidyverse: dplyr
Conflicts with tidy packages ---------------------------------------------------
filter(): dplyr, stats
lag():    dplyr, stats


Reading data using the read_csv function

In [5]:
myData<-read_csv("HW01pb1data.csv", col_names=FALSE)

Parsed with column specification:
cols(
  X1 = col_integer(),
  X2 = col_integer(),
  X3 = col_integer(),
  X4 = col_integer(),
  X5 = col_integer()
)
Warning message:
“2 parsing failures.
row col   expected      actual
405  X4 an integer thirty five
531  X5 an integer twenty five
”

Let's explicitly find out if there were some problematic entries in the data, even though read_csv gave us some warning above.

In [6]:
problems(myData)

row col expected   actual     
1 405 X4  an integer thirty five
2 531 X5  an integer twenty five

As seen above, rows 405 and 531 have incosistent data types in columns X4 and X5 respectively.